core/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -59,6 +60,19 @@
     public static final String INDEX_TRANSLOG_DURABILITY = "index.translog.durability";
```

Comment:
I will rename those all anyway in followups when I move it over to the new settings infra